src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.data.tag;
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+	private Person person;
+	private Tag tag;
+	private enum AddOrDelete{ADD,DELETE};
+	private AddOrDelete action;
+	
+	private final String ADD = "ADD";
```

Comment:
actually I just realised the problem here. You could have made AddOrDelete enum be shared between AddressBook and Tagging.